birdhouse/config/geoserver/config/proxy/conf.extra-service.d/geoserver.conf.template

```diff
@@ -14,6 +14,7 @@
 
     location = /secure-geoserver-auth {
         internal;
+        ${GEOSERVER_SKIP_AUTH_PROXY_INCLUDE}
```

Comment:
Can you add a space after this and a comment before the following `proxy_pass` that highlights that this part is skipped or nor according to `GEOSERVER_SKIP_AUTH` value? This can help figure out what is happening if looking at the resulting file after template variables were applied.